# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/editing/other/insertparagraph-with-white-space-style.tentative.html.ini
# Commit: 554d63dad12e
# Full Hash: 554d63dad12e4d1f2560e99de1fafac9a97da026
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2023-09-27 09:02:41
# Regressor Bug: 1851951
# File Overlap Count: 1
# Description:
#   Bug 1851951 - Make `HTMLEditor` refer computed `display` instead of the HTML default style at considering block or inline element r=m_kato
#   
#   Blink [1] and WebKit [2] refers `display-outside` value when they consider whether an
#   element is a block or an inline.
#   
# ==============================================================================

diff -r 97dc7be3b93c -r 554d63dad12e testing/web-platform/meta/editing/other/insertparagraph-with-white-space-style.tentative.html.ini
--- a/testing/web-platform/meta/editing/other/insertparagraph-with-white-space-style.tentative.html.ini	Wed Sep 27 02:50:33 2023 +0000
+++ b/testing/web-platform/meta/editing/other/insertparagraph-with-white-space-style.tentative.html.ini	Wed Sep 27 03:01:26 2023 +0000
@@ -260,19 +260,24 @@
   expected:
     if (os == "android") and fission: [OK, TIMEOUT]
   [<div contenteditable style="white-space:nowrap; display:inline">abc[\]</div> (defaultparagraphseparator: div)]
-    expected: FAIL
+    expected:
+      if not early_beta_or_earlier: FAIL
 
   [<div contenteditable style="white-space:nowrap; display:inline">[\]abc</div> (defaultparagraphseparator: div)]
-    expected: FAIL
+    expected:
+      if not early_beta_or_earlier: FAIL
 
   [<div contenteditable style="white-space:nowrap; display:inline">a[\]bc</div> (defaultparagraphseparator: div)]
-    expected: FAIL
+    expected:
+      if not early_beta_or_earlier: FAIL
 
   [<div contenteditable style="white-space:nowrap; display:inline">abc[\]</div> (defaultparagraphseparator: div) (preserving temporary inline style test)]
-    expected: FAIL
+    expected:
+      if not early_beta_or_earlier: FAIL
 
   [<div contenteditable style="white-space:nowrap; display:inline"><b>abc[\]</b></div> (defaultparagraphseparator: div) (preserving inline style test)]
-    expected: FAIL
+    expected:
+      if not early_beta_or_earlier: FAIL
 
   [<div contenteditable><div style="display:inline; white-space:nowrap">abc[\]</div></div> (defaultparagraphseparator: div)]
     expected: FAIL
@@ -293,19 +298,24 @@
     expected: FAIL
 
   [<div contenteditable style="white-space:nowrap; display:inline-block">abc[\]</div> (defaultparagraphseparator: div)]
-    expected: FAIL
+    expected:
+      if not early_beta_or_earlier: FAIL
 
   [<div contenteditable style="white-space:nowrap; display:inline-block">[\]abc</div> (defaultparagraphseparator: div)]
-    expected: FAIL
+    expected:
+      if not early_beta_or_earlier: FAIL
 
   [<div contenteditable style="white-space:nowrap; display:inline-block">a[\]bc</div> (defaultparagraphseparator: div)]
-    expected: FAIL
+    expected:
+      if not early_beta_or_earlier: FAIL
 
   [<div contenteditable style="white-space:nowrap; display:inline-block">abc[\]</div> (defaultparagraphseparator: div) (preserving temporary inline style test)]
-    expected: FAIL
+    expected:
+      if not early_beta_or_earlier: FAIL
 
   [<div contenteditable style="white-space:nowrap; display:inline-block"><b>abc[\]</b></div> (defaultparagraphseparator: div) (preserving inline style test)]
-    expected: FAIL
+    expected:
+      if not early_beta_or_earlier: FAIL
 
   [<div contenteditable><div style="display:inline-block; white-space:nowrap">abc[\]</div></div> (defaultparagraphseparator: div)]
     expected: FAIL
@@ -326,19 +336,24 @@
     expected: FAIL
 
   [<div contenteditable style="white-space:nowrap; display:inline">abc[\]</div> (defaultparagraphseparator: p)]
-    expected: FAIL
+    expected:
+      if not early_beta_or_earlier: FAIL
 
   [<div contenteditable style="white-space:nowrap; display:inline">[\]abc</div> (defaultparagraphseparator: p)]
-    expected: FAIL
+    expected:
+      if not early_beta_or_earlier: FAIL
 
   [<div contenteditable style="white-space:nowrap; display:inline">a[\]bc</div> (defaultparagraphseparator: p)]
-    expected: FAIL
+    expected:
+      if not early_beta_or_earlier: FAIL
 
   [<div contenteditable style="white-space:nowrap; display:inline">abc[\]</div> (defaultparagraphseparator: p) (preserving temporary inline style test)]
-    expected: FAIL
+    expected:
+      if not early_beta_or_earlier: FAIL
 
   [<div contenteditable style="white-space:nowrap; display:inline"><b>abc[\]</b></div> (defaultparagraphseparator: p) (preserving inline style test)]
-    expected: FAIL
+    expected:
+      if not early_beta_or_earlier: FAIL
 
   [<div contenteditable><div style="display:inline; white-space:nowrap">abc[\]</div></div> (defaultparagraphseparator: p)]
     expected: FAIL
@@ -359,19 +374,24 @@
     expected: FAIL
 
   [<div contenteditable style="white-space:nowrap; display:inline-block">abc[\]</div> (defaultparagraphseparator: p)]
-    expected: FAIL
+    expected:
+      if not early_beta_or_earlier: FAIL
 
   [<div contenteditable style="white-space:nowrap; display:inline-block">[\]abc</div> (defaultparagraphseparator: p)]
-    expected: FAIL
+    expected:
+      if not early_beta_or_earlier: FAIL
 
   [<div contenteditable style="white-space:nowrap; display:inline-block">a[\]bc</div> (defaultparagraphseparator: p)]
-    expected: FAIL
+    expected:
+      if not early_beta_or_earlier: FAIL
 
   [<div contenteditable style="white-space:nowrap; display:inline-block">abc[\]</div> (defaultparagraphseparator: p) (preserving temporary inline style test)]
-    expected: FAIL
+    expected:
+      if not early_beta_or_earlier: FAIL
 
   [<div contenteditable style="white-space:nowrap; display:inline-block"><b>abc[\]</b></div> (defaultparagraphseparator: p) (preserving inline style test)]
-    expected: FAIL
+    expected:
+      if not early_beta_or_earlier: FAIL
 
   [<div contenteditable><div style="display:inline-block; white-space:nowrap">abc[\]</div></div> (defaultparagraphseparator: p)]
     expected: FAIL